### DISCUSSION TITLE
need help my friends !!!!!

### DISCUSSION THREAD

**Discussion by sureshpandiyan1** [community/community#36579] [Other Feature Feedback, Questions, & Ideas]
if somebody got banned using my scraper, even it's unlicense
is there owner of repo can responsible for that ? 

seriously, i dont want to lose my repos :(

**Comment by haseeb-xd** [comment]
I don't think you will be responsible for that because there are alot of hacking tools which are open source and only for educational purposes.

**Reply by sureshpandiyan1** [reply]
Thank u for reply too me... Someone blackmail me to get my GitHub taken down... Even it's free license

**Comment by ossanna16** [comment]
Hey there @sureshpandiyan1 and thank you for asking a good question! I'm glad you found our community 🙂
